src/pages/ProfilePage/components/profile-card/index.tsx

```diff
@@ -0,0 +1,35 @@
+import LogoutButton from '@/pages/ProfilePage/components/logout-button';
+import ProfileImage from '@/pages/ProfilePage/components/profile-image';
```

Comment:
```suggestion
import ProfileImage from './components/profile-image';
```

하위 컴포넌트이기 때문에 상대경로로 적어주는 것이 조금 더 이해가 쉬운 것 같아요.